astropy/time/core.py

```diff
@@ -573,6 +573,9 @@ def light_travel_time(self, skycoord, kind='barycentric', location=None):
             The location of the observatory to calculate the correction for.
             If no location is given, the ``location`` attribute of the Time
             object is used
+        ephemeris : str, optional
+            Solar system ephemeris to use. By default, use the one set with
```

Comment:
I realise this is a copy from what we use inside `solar_system`, but we don't have the ephemeris note here. Maybe at least give some examples, say
```
Solar system ephemeris to use (e.g., 'builtin', 'jpl'). By default, use the
one set with ``astropy.coordinates.solar_system_ephemeris.set``.
For more details, see `<link to documentation you wrote>`_.
```